test/common/CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ TARGET_COMPILE_OPTIONS(
     PUBLIC ${_COMMON_COMPILE_OPTIONS_PUBLIC})
 TARGET_LINK_LIBRARIES(
     ${_COMMON_TARGET_NAME}
-    PUBLIC alpaka ${_COMMON_LINK_LIBRARIES_PUBLIC})
+    PUBLIC "alpaka;${_COMMON_LINK_LIBRARIES_PUBLIC}")
```

Comment:
I looked again into the [cmake documentation](https://cmake.org/cmake/help/latest/command/target_link_libraries.html) and it looks like this must be separated by spaces.
I will change it.

Reply:
CMake is allowing a lists and space separated entries. Never the less I will change it to separate entries.